src/librbd/image/RemoveRequest.h

```diff
@@ -0,0 +1,127 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_LIBRBD_IMAGE_REMOVE_REQUEST_H
+#define CEPH_LIBRBD_IMAGE_REMOVE_REQUEST_H
+
+#include "include/int_types.h"
+#include "include/rbd/librbd.hpp"
```

Comment:
Double-check these includes -- I think a lot are not required in this header file since I don't see them being used.